source/docs/software/wpilib-tools/robot-characterization/using-toolsuite.rst

```diff
@@ -0,0 +1,62 @@
+Generating a Project
```

Comment:
This file seems to be a duplicate of generating-project.rst?

Reply:
[WIP]